web/main/models.py

```diff
@@ -942,14 +951,12 @@ def _search(
                 if order_by in ["created_at", "effective_date"]:
                     order_by_expression = ["-metadata__effective_date", display_name]
         results = results.order_by(*order_by_expression)
-        results = Paginator(results, page_size).get_page(page)
+        paged_results = Paginator(results, page_size).get_page(page)
```

Comment:
This is that pattern where mypy got mad because we've reassigned a variable to an object of a different type.